### ISSUE TITLE
requirejs.config  "map" no working

### ISSUE THREAD

**Issue by acelan86** [smysnk/gulp-rev-all#57]
when my require config seems:

```javascript
requirejs.config({
    baseUrl: "./",
    paths: {
        react: "lib/react",
        jquery: "lib/jquery"
    },
    "shim": {
        "jquery" : {
            "exports": "$"
        }
    },
    map: {
        "*": {
            css: "lib/css"
        }
    }
});
```

"map" in config is no match   amdConfigRegex

**Comment by circlingthesun** [comment]
The current implementation only reads up to the path section and searches for references inside. The map section is never considered. PR's are welcome ;)

**Comment by smysnk** [comment]
Should be fixed in v0.8.0, please give it a try.
